dbcleanup_plugin.py

```diff
@@ -323,24 +322,24 @@ def export_cleaned_records(
 
 # Creating a flask appbuilder BaseView
 class AstronomerDbcleanup(AppBuilderBaseView):
+    resource_name = "dbcleanup"
     default_view = "dbcleanup"
+    csrf_exempt = False
+    base_permissions = ["can_access_dbcleanup"]
+    # A bug in Flask-AppBuilder mandates us to have a value for method_permission_name
+    # before is_item_public filter can be used.
+    method_permission_name = {
+        "access_dbcleanup": "access_dbcleanup",
+    }
```

Comment:
I don't think you need this though. It was needed in version check plugin because we used is_item_public here: https://github.com/astronomer/astronomer-airflow-version-check/blob/9c9754dfea36de88a9a4882aa016164cb98dfd16/astronomer/airflow/version_check/templates/update-available.html#L4